tests/unit/test_image_agnostic_util.py

```diff
@@ -20,15 +20,15 @@ def test_generate_dockerfile():
 def test_get_new_image_name_legacy():
     # test non-eventstream runtime (sandbox-based)
     base_image = 'debian:11'
-    new_image_name = _get_new_image_name(base_image, is_eventstream_runtime=False)
+    new_image_name = _get_new_image_name(base_image)
     assert new_image_name == 'od_sandbox:debian__11'
 
     base_image = 'ubuntu:22.04'
```

Comment:
Is 23.x considered as well? Might shave off some time during build?

Reply:
Ugh.. this is test script, sorry, didn't find the right location to put my comment! 😂

Reply:
What do you mean by "23.x" - is that a particular version of the image?

Reply:
Yes, isn't there a Ubuntu 23.x image instead of 22 maybe?

Reply:
ah yes! So this unit test is more like testing the parsing / renaming part of the agnostic image. In the next PR, I'm trying to write test cases like `test_sandbox.py` that actually tests the usability of these Runtime on different images.

Reply:
Maybe we could switch our other images from 22 to 23 in general, I put my comment in a bad place, it wasn't about the test script itself, basically. Sorry! 😂